ledger-core/v2/testutils/slotdebugger/stepmachine.go

```diff
@@ -56,7 +58,7 @@ func New(ctx context.Context, t *testing.T, suppressLogError bool) *StepControll
 		ScanCountLimit:    1e5,
 		LogAdapterCalls:   true,
 		SlotMachineLogger: debugLogger,
-		SlotAliasRegistry: &conveyor.GlobalAliases{},
+		SlotAliasRegistry: &slotAliasRegistry,
```

Comment:
thats actually bad, because now two tests that create stepcontroller for each of them can interefere each other. Why do you need global registry?

Reply:
Fixed.